### DISCUSSION TITLE
text in footer svg

### DISCUSSION THREAD

**Discussion by ghost** [chillerlan/php-qrcode#212] [Q&A]
Hello, great job

I know imageWithText.php so
my question
can add text below qr code in svg format?
Thanks

**Comment by codemasher** [comment]
Hi! basically you just need to add a `<text>` node to the SVG (see https://developer.mozilla.org/en-US/docs/Web/SVG/Element/text) and adjust the viewbox size. Here's a simple example where a node is added to the SVG in a lazy way: https://github.com/chillerlan/php-qrcode/blob/main/examples/svgWithLogoAndCustomShapes.php#L34-L49  - basically just replace the closing `</svg>` with `$myTextNode.'</svg>'`.

Alternatively, you can extend the `QRMarkupSVG` class (the clean way) [and add a method](https://github.com/chillerlan/php-qrcode/blob/main/examples/svgWithLogo.php#L32-L42) that generates the text node and [overwrite the `header()` method](https://github.com/chillerlan/php-qrcode/blob/main/src/Output/QRMarkupSVG.php#L75-L95) to adjust the viewbox accordingly.
